src/main/java/org/icgc/argo/program_service/services/ego/EgoService.java

```diff
@@ -365,4 +365,14 @@ public EgoUser getOrCreateUser(
   public EgoUser convertInvitationToEgoUser(@NonNull JoinProgramInviteEntity invite) {
     return programConverter.joinProgramInviteToEgoUser(invite);
   }
+
+  public boolean isUserDacoApproved(@Email String email) {
+    val user = egoClient.getUser(email).orElse(null);
+    if (user == null) {
+      return false;
+    }
+    val userGroups = egoClient.getGroupsByUserId(user.getId());
+    return userGroups.anyMatch(
+        group -> group.getName().equals("DACO") && group.getStatus().equals("APPROVED"));
```

Comment:
DACO access isn't done through groups anymore... We are instead looking for a scope on the user, specifically DACO.READ or DACO.WRITE .

Reply:
Second change needed here, there are two hardcoded strings. These should be managed in a common constants file instead of scattered throughout the code and then imported for use here.

Reply:
@joneubank, I updated the daco appoved check. Also I moved the hardcoded strings to app properties so they can be configured.